docs/components/novatalkscore.md

```diff
@@ -9,6 +9,22 @@
 ##Product Notices
 ***
 ##2022R4
+
+###2022R4-5 :briefcase: Epic
+####New Features
+
+- *NovaTalks.Core|UI*: Calendars [:clipboard: NC2-372] [NC2-372]
```

Comment:
"Core" OR "UI" ? This is not a script language.
Write this task in both UI and Core md files, and follow documentation format.